src/Symfony/Component/Notifier/Bridge/Telegram/composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": ">=8.1",
         "symfony/http-client": "^5.4|^6.0|^7.0",
-        "symfony/notifier": "^6.2.7|^7.0"
+        "symfony/notifier": "^6.4|^7.0"
```

Comment:
This would have been caught by the prefer-lowest job if the tests of this bridges were including a test covering the validation of exclusive options. This is the test that should be added IMO, more than a test of the exception constructor in the component.

Reply:
You are right, will do it